webCommander/centralHub/views.py

```diff
@@ -1,9 +1,10 @@
 from django.http import HttpResponse, JsonResponse
```

Comment:
there is a bunch of unused imports, you can check with flake8 or use pyflakes (in ALE) https://apythonistalearningrust.com/rust-analyzer-ale-errors-while-coding/